tools/trap/HowToUseTrap.md

```diff
@@ -262,32 +262,91 @@ Ramdump received successfully..!
 ```
 
 ## How to parse RAMDUMP
-DumpParser Script provides two interfaces: CUI and GUI
+TRAP Script provides two interfaces: CUI and GUI
 
-### DumpParser using CUI
+### TRAP using CUI
 
 #### To get call stack using RAM dump
-1. Run Ramdump Parser Script
+1. Change the directory
 ```
 cd $TIZENRT_BASEDIR/tools/trap/
 ```
-2. See the Output
+2. Copy crash logs  
+    First copy the crash logs to a file in tools/trap/`<log_file>`
```

Comment:
Do we need this 2nd step even this tab is `To get call stack using RAM dump`? If it is, why do you guide two different tabs with the same content? And what can we do with ramdump? I mean why do we need ramdump?

Reply:
I think we can merge the two under one menu
"To get call stack"
Ramdump is used to retrieve call stack for scenarios when crash logs are not accessible and for the gui_ramdumpParser.py script.

Reply:
Are you still modifying? There is no change on this.